changelogs/fragments/sub_plugins_dir_change.yaml

```diff
@@ -0,0 +1,5 @@
+---
+breaking_changes:
+  - Move sub plugins cli_parsers, fact_diff and validate to `plugins/sub_plugins` folder
+  - If added custom sub plugins in your collection move from old location `plugins/<sub-plugin-name>` to the
+    new location `plugins/sub_plugins/<sub-plugin-name>` and update the imports as required
```

Comment:
In case of `cli_parsers`, `<sub-plugin-name>` changes from `cli_parsers` to `cli_parser`. This is probably something worth mentioning here as well.